### DISCUSSION TITLE
Chia 1.2.11 released

### DISCUSSION THREAD

**Discussion by sargonas** [Chia-Network/chia-blockchain#9139] [Announcements]
We've released v. 1.2.11!

This release includes support for new plotters (MadMax and Bladebit) as well as other features, and most importantly includes several optimizations based on data gathered from the Dust Storm.

The full changelog includes links to how to utilize these new plotters, as well as details on the optimizations we have made: https://github.com/Chia-Network/chia-blockchain/releases/tag/1.2.11

(As always, builds take a while to compile so please wait up to 30 minutes from the time of this post if the installer you are looking for isn't listed yet!)

**Comment by lekman1** [comment]
Hello,
I updated my Chia Blockchain and he can not synchronize.
Please help what to do, I tried everything.
Thank you in advance,

**Reply by loppefaaret** [reply]
come on over to https://github.com/Chia-Network/chia-blockchain/discussions/categories/support and create your own thread, and ill try and help you out best i can - this is just not the place to start supporting you with this issue
you might even add @loppefaaret to your post, so that i get a notification

**Comment by rmica** [comment]
I highly appreciate the "full" database sync option. Now I can rsync data to a backup box without problems. Thanks.
